api/src/main/java/org/openmrs/module/patientflags/api/FlagService.java

```diff
@@ -259,6 +342,15 @@
 	@Transactional(readOnly = true)
 	@Authorized(value = { "Manage Flags", "Test Flags" }, requireAll = false)
 	public DisplayPoint getDisplayPoint(Integer displayPointId);
+	/**
+	 * Retrieve a specific DisplayPoint based on its uuid
+	 *
+	 * @param uuid
+	 * @return the DisplayPoint which matches the specific uuid
+	 */
+	@Transactional(readOnly = true)
+	@Authorized(value = { "Manage Flags", "Test Flags" }, requireAll = false)

```

Comment:
Did we agree to replace the "Test Flags" privilege?

Reply:
@dkayiwa this will be done in a separate PR